src/utils/getPath.ts

```
@@ -5,8 +1,53 @@
+  const slug = blogId.length > 0 ? blogId.slice(-1) : blogId;
+
+  // If not inside the sub-dir, simply return the file path
+  if (!pathSegments || pathSegments.length < 1) {
+    return [basePath, slug].join("/");
   }
-  return `/${data.contentType}/${slug}`;
-};
 
-export default getRelativePath;
+  const res = [basePath, ...pathSegments, slug].join("/");
+  console.log("getPath", url, res);
```

ESLint

[error] Unexpected console statement.

(no-console)